src/nsrt_learning/sampler_learning.py

```diff
@@ -169,7 +188,8 @@ def _create_sampler_data(
     preconditions: Set[LiftedAtom], add_effects: Set[LiftedAtom],
     delete_effects: Set[LiftedAtom], param_option: ParameterizedOption,
     datastore_idx: int
-) -> Tuple[List[Tuple[State, Dict[Variable, Object], _Option]], ...]:
+) -> Tuple[List[Tuple[State, Dict[Variable, Object], _Option,
+                      Optional[Set[GroundAtom]]]], ...]:
```

Comment:
This is getting a little crazy... let's maybe add this to bottom of structs.py?
`SamplerDatapoint = Tuple[State, Dict[Variable, Object], _Option, Optional[Set[GroundAtom]]]`

and then we can change this return type to:
`Tuple[List[SamplerDatapoint], List[SamplerDatapoint]]`
which would further eliminate the unnecessarily vague use of `...` here.

Reply:
done, thanks!